docs/user-guide/kubectl/kubectl_apply.md

```diff
@@ -41,6 +41,7 @@ Apply a configuration to a resource by filename or stdin
 
 Apply a configuration to a resource by filename or stdin.
 The resource will be created if it doesn't exist yet.
+If you want to use apply, please always use apply to create the resource initially or use create --save-config.
```

Comment:
Suggest rephrase it to -- `To use 'apply', always create the resource initially with either 'apply' or 'create --save-config'.`